src/components/user_bids_page/UserBidCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ItemCard from 'components/item_card/ItemCard';
+import './UserBidCard.scss';
+import { dateFormatter } from 'utilities/date';
+import { pennyToDollarConverter } from 'utilities/price';
+
+const UserBidCard = ({ item, bids }) => {
+    return (
+        <section className='user-bid--container'>
+            <ItemCard item={item} showDescription={true} />
+            <section className='user-bids--grid'>
+                {bids && bids.map(bid => {
+                    return <UserBid bid_price={bid.bid_price} timestamp={bid.timestamp} />
+                })}
+            </section>
+        </section>
+    )
+}
+
+const UserBid = ({ timestamp, bid_price }) => (
+    <section className='bid--container'>
+        <div className='bid--container-info'>
+            <h2>You Bid</h2>
+            <h5>{dateFormatter(timestamp)}</h5>
+        </div>
+        <h1 className='bid--container-price'>${pennyToDollarConverter(bid_price)}</h1>
```

Comment:
Nitpick - I prefer to name functions as verbs, not nouns - so this would be something like `convertPenniesToDollars`

Reply:
oK! I'll change this in the most recent PR branch (as they are all chained together, and pushing a change here will require numerous rebases for each child branch :D )